INSTALL.md

```diff
@@ -81,13 +81,35 @@ $ dpkg -i mycms*.deb
 #### Dependencies
 
 ```
-$ sudo apt install nsis
+$ sudo apt install mingw-w64 nsis
 ```
 
 #### Build
 
 ```
-$ ./configure --host=x86-w64-mingw32 --prefix=/ ...
-$ make install DESTDIR="$(pwd)/tmp"
-$ DESTDIR=tmp ./packaging/windows-nsis/build
+$ set MYCMS_PREFIX=/tmp/mycms-prefix
+```
+
+#### OpenSSL Build
+
+Download and extract OpenSSL tarball.
+
+Run the following in OpenSSL source directory:
+
+```
+$ ./Configure --prefix="${MYCMS_PREFIX}" --cross-compile-prefix=x86_64-w64-mingw32- mingw64
```

Comment:
/bin/bash ../../libtool --mode=compile --tag=RC  -DHAVE_CONFIG_H -I. -I../..  -I../../include  -DWIN32_LEAN_AND_MEAN -i "versioninfo.rc" -o "versioninfo.lo"
Usage: /home/g/tmp/tmp1/mycms/libtool [OPTION]... [MODE-ARG]...
Try 'libtool --help' for more information.
libtool:   error: unrecognised option: '-DHAVE_CONFIG_H'

Reply:
Have you installed `mingw-w64`? can you please send me config.log?